jet_tracking/context.py

```diff
@@ -1,20 +1,90 @@
 import json
 import logging
 import os
-import threading
 from argparse import ArgumentParser
 from pathlib import Path
 
 import numpy as np
 import yaml
 
-log = logging.getLogger(__name__)
-lock = threading.Lock()
+log = logging.getLogger("jet_tracker")
```

Comment:
the `__name__` logger in the previous version of the code is the canonical way to do this, by dropping this you're actually losing context information about where the log call was made

Reply:
Good to know. Thanks